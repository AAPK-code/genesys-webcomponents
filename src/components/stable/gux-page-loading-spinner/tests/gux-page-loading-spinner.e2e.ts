import { newE2EPage } from '@stencil/core/testing';

describe('gux-page-loading-spinner', () => {
  it('renders', async () => {
    const page = await newE2EPage();

    await page.setContent(
      '<gux-page-loading-spinner></gux-page-loading-spinner>'
    );

    const element = await page.find('gux-page-loading-spinner');

    expect(element.outerHTML).toMatchSnapshot();
  });
});
