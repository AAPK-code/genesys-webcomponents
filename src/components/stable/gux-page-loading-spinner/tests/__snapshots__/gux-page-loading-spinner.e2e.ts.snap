// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-page-loading-spinner renders 1`] = `"<gux-page-loading-spinner class=\\"hydrated\\"><div class=\\"page-loading-spinner gux-page-loading-animation\\"></div></gux-page-loading-spinner>"`;
